src/opm/common/utility/numeric/calc_cellvol.cpp

```diff
@@ -0,0 +1,199 @@
+#include <opm/common/utility/numeric/calc_cellvol.hpp>
+
+
+/* 
+    Cell volume calculation based on following publication:
+   
+    D. K Pointing, Corner Point Geometry in Reservoir Simulation , 
+    ECMOR I - 1st European Conference on the Mathematics of Oil Recovery, 
+    1989 
+*/
+ 
+using namespace std;
+
+
+
+
+double C(const std::vector<double>& r,int i1,int i2,int i3){
```

Comment:
I have some trouble understanding what C computes. Please add a comment giving some information, a formula or similar. If it is named to match the paper (I don't have that in front of me) giving an equation number from the paper is a good idea. If possible, find a better name... Also in OPM capitalized names are usually reserved for classes or types.

Some minor nitpicks: add space after the commas, before the opening brace (or start the function body on the next line), and around binary operators, in particular assigment. This applies throughout.

Reply:
>  If possible, find a better name... Also in OPM capitalized names are usually reserved for classes or types.

I have used capital C as name on this function because I'm following the nomenclature used in the paper. Please advise. Would you like me to change this ?